starter-code/js/index.js

```diff
@@ -1,21 +1,32 @@
 function deleteItem(e){
+   console.log(e);
+   del = document.getElementsByClassName('content2');
+   divs = document.getElementsByTagName('div');
+   del.removeChild(divs);
```

Comment:
Be careful! If this worked, It would delete ALL the divs inside content2 (not only one).